nucliadb/src/nucliadb/ingest/orm/brain.py

```diff
@@ -301,7 +301,11 @@ def apply_field_vectors(
 
         if replace_field:
             full_field_id = ids.FieldId(rid=self.rid, type=fid.type, key=fid.key).full()
-            self.brain.sentences_to_delete.append(full_field_id)
+            if vectorset is None:
+                # DEPRECATED
```

Comment:
should we add a warning here? or will it be too noisy because we still have KBs on prod without vectorsets?

Reply:
Not sure, but I'd add a warning the day we want to finally remove the default vectorset concept